### PR TITLE
added CODEOWNERS file

### PR DESCRIPTION
**Pull request by nadongguri**
Added CODEOWNERS file to define teams are responsible for code in a repo.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Lines starting with '#' are comments.
```
Comment (romandev) [inline CODEOWNERS]: I'm not sure if these comments are necessary..
Could you please remove these or replace these with new comments to fit our situation.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*       @romandev
+
+# Order is important. The last matching pattern has the most precedence.
```
Comment (romandev) [inline CODEOWNERS]: ditto

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
```
Comment (romandev) [inline CODEOWNERS]: ditto

**Comment by nadongguri** [comment]
got it!

**Comment by nadongguri** [comment]
@romandev, removed all comments. :-)

**Pull request merged**